website/static/css/fangorn.css

```diff
@@ -37,6 +37,10 @@ color: #F00;
 text-align: center;
 }
 
+#treeGrid span {
```

Comment:
It's generally a bad idea to use IDs for CSS rules (but not always) because it's brittle. Not all of our treebeard grids have the idea `#treeGrid`. Can you use a different selector?